pkgs/desktops/gnome/core/gnome-keyring/default.nix

```diff
@@ -63,6 +63,9 @@ stdenv.mkDerivation rec {
     # gnome-keyring doesn't build with ssh-agent by default anymore, we need to
     # switch to using gcr https://github.com/NixOS/nixpkgs/issues/140824
     "--enable-ssh-agent"
+    "LIBGCRYPT_CONFIG=${lib.getExe' (lib.getDev libgcrypt) "libgcrypt-config"}"
```

Comment:
Please mention these are for cross in the comment or they will be likely removed in the future.